.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+Closes
+
+# Title
+
+## Contributing Guidelines
+
+If you haven't, go over the Contributing Guidelines.
+
+> Put an `x` in the box if you did.
+
+- [ ] I have read the **CONTRIBUTING** document.
+
+## Related Issue
+
+**This project only accepts pull requests related to open issues.**
+
+If suggesting a new feature or change, please discuss it in an issue first.
+
+> Please link to the issue here.
+
+## Description
+
+> You may refer to the original issue if it has enough context.
+
+> If anything changed since the original issue was logged, make a note of it here.
```

Comment:
Just to make sure I understood, this is the template for what people should write on the description of the PR, right?
Should we also have guidelines for the title of the PR? In particular:
- should the title of the PR be the same as the issue's?
- should the title of the PR refer to the issue id? Or is it enough that the branch name does?
- should the PR title have some "state information"? Like adding a [WIP] while the PR is being worked on, and an [RFC] once it's ready for comments, or something similar?

Reply:
Great points. All of the above, I think.